repositories/src/test/scala/cromwell/pipeline/datastorage/dao/utils/PostgreTestSources.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.pipeline.datastorage.dao.utils
+
+import com.dimafeng.testcontainers.{ ForAllTestContainer, PostgreSQLContainer }
+import com.typesafe.config.Config
+import cromwell.pipeline.datastorage.DatastorageModule
+import cromwell.pipeline.utils.{ ApplicationConfig, TestContainersUtils, TestTimeout }
+import org.scalatest.{ AsyncWordSpec, BeforeAndAfterAll, BeforeAndAfterEach }
+import slick.jdbc.PostgresProfile.api._
+
+import scala.concurrent.Await
+
+trait PostgreTestSources
+    extends AsyncWordSpec
+    with ForAllTestContainer
+    with BeforeAndAfterEach
+    with BeforeAndAfterAll
+    with TestTimeout {
+
+  override val container: PostgreSQLContainer = TestContainersUtils.getPostgreSQLContainer()
+  private lazy val config: Config = TestContainersUtils.getConfigForPgContainer(container)
+  lazy val datastorageModule: DatastorageModule = new DatastorageModule(ApplicationConfig.load(config))
```

Comment:
`dataStorageModule` looks better ;)

Reply:
I genuinely thought you liked smth and never paid attention to the different camelcasing in your message, sry :)
Should I change it? Thought that that val was named accordingly to camelcasing of it's type

Reply:
Actually doesn't matter, it was intellij suggestion, at your discretion